sketch_oct9a.ino

```diff
@@ -324,6 +343,7 @@ void setup() {
 
   Scheduler.startLoop(gps_loop);
   Scheduler.startLoop(radio_loop);
+  Scheduler.startLoop(save_loop);
```

Comment:
I understand this is necessary, but I feel like we should closely watch/look at how many different loops we're starting on a single-core system. If we put the 'saveToSD' stuff in the save loop as another, would it be quicker?

Reply:
Yes, I was able to make it a bit faster by merging it with the GPS loop, but still nowhere near as fast as it was before. It's possible that simply writing the data to the SD card is a bottleneck and we should be careful how/when we do it.

Reply:
I think you're looking into that already, so keep messing around with it and try and find what's best